virtualenvapi/manage.py

```diff
@@ -58,7 +61,10 @@ def _errorfile(self):
 
     def _create(self):
         """Executes `virtualenv` to create a new environment."""
-        proc = subprocess.Popen(['virtualenv', self.name], cwd=self.root, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        if self.python is None:
+            proc = subprocess.Popen(['virtualenv', self.name], cwd=self.root, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        else:
+            proc = subprocess.Popen(['virtualenv', '-p', self.python, self.name], cwd=self.root, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Could we move the `['virtualenv', ...]` bit out of the Popen and into the `self.python` conditional so we're not repeating the Popen call twice?